config.js

```diff
@@ -32,6 +32,13 @@ var conf = convict({
       env: 'HTTPDB_URL'
     }
   },
+  contentServer: {
+    url: {
+      doc: 'The url of the corresponding fxa-content-server instance',
+      default: 'https://accounts.firefox.com',
```

Comment:
nit: the auth-server uses ` default: 'http://127.0.0.1:3030'` for this, may be worth doing the same for full consistency.  I don't feel strongly either way.